README.md

```diff
@@ -175,46 +193,51 @@ After downloading/installing the prerequisites, ensure that you can execute the
 
-You can then run the system tests by issuing the following command:
+Run the system tests:
 
 ```
 npm run test:system
 ```
 
 <br/>
 
-**IMPORTANT!** Do not commit configured properties files because they contain security principles and other critical information.
+**IMPORTANT!** Do not commit configured properties files to this repository because they contain security principles and other critical information.
 
 <br/>
 
-## **Frequently Asked Questions**
+## Frequently asked questions
 
-**How can I install Zowe CLI as a root user on Mac/Linux?**
+### How can I install Zowe CLI as a root user on Mac/Linux?
 
-  - You can install the CLI as root so that all users can access the CLI without installing it individually on their user account. As the root user on Mac/Linux, issue the following command:
+  - Install the CLI as root so that all users can access the CLI without installing it individually on their user account. As the root user on Mac/Linux, issue the following command:
 
     ```
     npm i -g @zowe/cli@latest --ignore-scripts
     ```
+
     **WARNING!** If you use this method, plug-ins that are installed as root can only be accessed as root. Users must install plug-ins on their user account or share all profiles/plugins/settings/logs with root. You also might encounter npm errors if you install as root. We recommend that Linux administrators implement a user/group environment where permissions can be more carefully controlled.
 
-**What is the difference between V1 and V2?**
+### What is the difference between Zowe V2 and V3?
 
-  - V2 uses **team profiles** and **deprecates the Secure Credential Store** (SCS) plug-in.
+  - V2 introduces **team profiles** and **deprecates the Secure Credential Store** (SCS) plug-in used in Zowe V1.
 
-    Connection details can be managed efficiently within one file, promoting a global configuration that can be shared across teams and mainframe services. For more information on how to use profiles, visit [Zowe Docs](https://docs.zowe.org/stable/user-guide/cli-using-using-team-profiles/).
+    - Connection details can be managed efficiently within one file, promoting a global configuration that can be shared across teams and mainframe services. For more information on how to use profiles, see [Team configurations](https://docs.zowe.org/stable/user-guide/cli-using-using-team-profiles/) in Zowe Docs.
     
-    Secure credential encryption is included in the core CLI.
+    - Secure credential encryption is included in the core CLI.
+
+  - V3 includes the preceding features. Additionally, deprecates support for Zowe V1 profiles.
```

Comment:
I would change to "... Additionally, removes support for Zowe V1 profiles." or something similar since "deprecated" implies that something still works but is no longer recommended.